axelrod/tests/unit/test_fingerprint.py

```diff
@@ -269,135 +267,156 @@ def test_plot_figure(self):
-            Point(x=0.0, y=0.75): 1.030,
-            Point(x=0.0, y=1.0): 0.980,
-            Point(x=0.25, y=0.0): 3.000,
-            Point(x=0.25, y=0.25): 2.130,
-            Point(x=0.25, y=0.5): 1.940,
-            Point(x=0.25, y=0.75): 2.060,
-            Point(x=0.25, y=1.0): 1.940,
-            Point(x=0.5, y=0.0): 3.000,
-            Point(x=0.5, y=0.25): 2.300,
-            Point(x=0.5, y=0.5): 2.250,
-            Point(x=0.5, y=0.75): 2.420,
-            Point(x=0.5, y=1.0): 2.690,
-            Point(x=0.75, y=0.0): 3.000,
-            Point(x=0.75, y=0.25): 2.400,
-            Point(x=0.75, y=0.5): 2.010,
-            Point(x=0.75, y=0.75): 2.390,
-            Point(x=0.75, y=1.0): 2.520,
-            Point(x=1.0, y=0.0): 3.000,
-            Point(x=1.0, y=0.25): 2.360,
-            Point(x=1.0, y=0.5): 1.740,
-            Point(x=1.0, y=0.75): 2.260,
-            Point(x=1.0, y=1.0): 2.260,
+            Point(x=0.0, y=0.0): 3.0,
+            Point(x=0.0, y=0.25): 1.6,
+            Point(x=0.0, y=0.5): 1.01,
+            Point(x=0.0, y=0.75): 1.04,
+            Point(x=0.0, y=1.0): 0.98,
+            Point(x=0.25, y=0.0): 3.0,
+            Point(x=0.25, y=0.25): 2.12,
+            Point(x=0.25, y=0.5): 1.81,
+            Point(x=0.25, y=0.75): 2.06,
+            Point(x=0.25, y=1.0): 1.86,
+            Point(x=0.5, y=0.0): 3.0,
+            Point(x=0.5, y=0.25): 2.4299999999999997,
+            Point(x=0.5, y=0.5): 2.37,
+            Point(x=0.5, y=0.75): 2.74,
+            Point(x=0.5, y=1.0): 2.68,
+            Point(x=0.75, y=0.0): 3.0,
+            Point(x=0.75, y=0.25): 2.4299999999999997,
+            Point(x=0.75, y=0.5): 1.8399999999999999,
+            Point(x=0.75, y=0.75): 2.34,
+            Point(x=0.75, y=1.0): 2.46,
+            Point(x=1.0, y=0.0): 3.0,
+            Point(x=1.0, y=0.25): 2.12,
+            Point(x=1.0, y=0.5): 1.8599999999999999,
+            Point(x=1.0, y=0.75): 2.0300000000000002,
+            Point(x=1.0, y=1.0): 2.26
         }
 
         af = axl.AshlockFingerprint(axl.GoByMajority, axl.TitForTat)
-        data = af.fingerprint(turns=50, repetitions=2, step=0.25, progress_bar=False)
+        data = af.fingerprint(turns=50, repetitions=2, step=0.25, progress_bar=False,
+                              seed=0)
 
         for key, value in data.items():
             self.assertAlmostEqual(value, test_data[key], places=2)
 
     @given(strategy_pair=strategy_lists(min_size=2, max_size=2))
-    @settings(max_examples=5)
+    @settings(max_examples=5, deadline=None)
     def test_pair_fingerprints(self, strategy_pair):
         """
         A test to check that we can fingerprint
         with any two given strategies or instances
         """
-        strategy, probe = strategy_pair
-        af = AshlockFingerprint(strategy, probe)
-        data = af.fingerprint(turns=2, repetitions=2, step=0.5, progress_bar=False)
-        self.assertIsInstance(data, dict)
+        # strategy, probe = strategy_pair
```

Comment:
why are these commented out @marcharper?

Reply:
They shouldn't be, it was temporary while debugging